client/my-sites/plans-features-main/index.jsx

```diff
@@ -97,6 +98,7 @@ const jetpackProducts = [
 			...JETPACK_BACKUP_PRODUCT_NAMES,
 		},
 		optionsLabel: 'Backup options',
+		upgradeProductMap: JETPACK_BACKUP_UPGRADE_MAP,
```

Comment:
Since this is already `product` context, let's ditch the `product`  part from the name. `upgradeMap` or `upgradePaths` sounds good enough to me.